neo4j-cli/aura/internal/subcommands/deployment/server/list.go

```diff
@@ -0,0 +1,92 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/neo4j/cli/common/clicfg"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/api"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/output"
+	"github.com/neo4j/cli/neo4j-cli/aura/internal/subcommands/deployment/utils"
+	"github.com/spf13/cobra"
+)
+
+func NewListCmd(cfg *clicfg.Config) *cobra.Command {
+	var (
+		organizationId string
+		projectId      string
+		deploymentId   string
+	)
+
+	const (
+		organizationIdFlag = "organization-id"
+		projectIdFlag      = "project-id"
+		deploymentIdFlag   = "deployment-id"
+	)
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Returns deployment servers",
+		Long:  "Returns servers for the given Fleet Manager deployment.",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := fmt.Sprintf("/organizations/%s/projects/%s/fleet-manager/deployments/%s/servers", organizationId, projectId, deploymentId)
+
+			cmd.SilenceUsage = true
+			resBody, statusCode, err := api.MakeRequest(cfg, path, &api.RequestConfig{
+				Method:  http.MethodGet,
+				Version: api.AuraApiVersion2,
+			})
+			if err != nil {
+				return err
+			}
+
+			if utils.IsSuccessful(statusCode) {
+				fields := []string{
```

Comment:
Are we sure we need to display all these fields in the table? Table output should be human readable, and, if possible, below 120 characters (the table in `list_test.go` is over 400 characters, which will pretty much be mangled in any display)